src/parking-violations-reporting/use-parking-violations.tsx

```diff
@@ -12,6 +12,11 @@ export class PermissionRequiredError extends Error {
     super('Permission required');
   }
 }
+export class NoLocationError extends Error {
+  constructor() {
+    super('Location missing');
```

Comment:
Should this be translated?

Reply:
It's not displayed to the user. It's used to differentiate the various error situations. The message is useful e.g. when debugging.